src/main/webapp/app/shared/sidebar/sidebar-accordion/sidebar-accordion.component.ts

```diff
@@ -67,6 +70,19 @@ export class SidebarAccordionComponent implements OnChanges, OnInit {
         }
     }
 
+    calculateTotalUnreadMessages(): void {
```

Comment:
I think the naming here is misleading, as you do not calculate the total unread messages, but the unread messages per group.